### PR TITLE
Setup RabbitMQ, gRPC, and messages for initial inventory roll out

### PR DESCRIPTION
**Pull request by doomspork**

### CODE REVIEW

File: lib/warehouse/application.ex [hunk 1]
```diff
@@ -10,8 +10,8 @@ defmodule Warehouse.Application do
   def start(_type, _args) do
     children = [
       Warehouse.Repo,
-      {GRPC.Server.Supervisor, {Warehouse.Endpoint, 50_051}},
-      {Warehouse.Broadway, []}
+      {GRPC.Server.Supervisor, {Warehouse.Endpoint, 50_052}}
+      # {Warehouse.Broadway, []}
```
Comment (doomspork) [inline lib/warehouse/application.ex:14]: This was committed by mistake :cold_sweat:

File: mix.exs [hunk 2]
```diff
@@ -34,14 +34,14 @@ defmodule Warehouse.MixProject do
...
-      {:appsignal, "~> 1.0"},
-      {:bottle, github: "system76/bottle", ref: "b3b78b6"},
+      {:bottle, github: "system76/bottle", ref: "0fc599f"},
       {:broadway_rabbitmq, "~> 0.6.0"},
       {:decimal, "~> 1.9.0", override: true},
       {:ecto_enum, "~> 1.4"},
       {:ecto_sql, "~> 3.5"},
       {:ex_aws_sqs, "~> 3.2"},
       {:ex_aws, "~> 2.1.6"},
+      {:flow, "~> 1.1"},
```
Comment (btkostner) [inline mix.exs:44]: Oh my. :eyes:

Comment (doomspork) [reply mix.exs:44]: I'm not going to use this. It was something I was trying when we were running into those issues before :grin:

Comment (btkostner) [reply mix.exs:44]: Ah alright. Wasn't sure if you were going to do something fancy

**Review by btkostner — APPROVED** [review]
Simple enough. We will probably need an rpc endpoint for calculating the total amount of a sku and if it needs to be reordered, but that can be done separately

**Comment by doomspork** [comment]
Yup! There's a couple of stats we'll want to expose as we move to this system. For now _some_ of that can still rely on HAL since we're using the same underlying database.

We probably want to think about adding the functionality to Assembly to calculate the demand for a particular SKU and/or component.

**Pull request merged**